### ISSUE TITLE
 ERROR from UglifyJs SyntaxError: Unexpected token: name (SocketProvider)

### ISSUE THREAD

**Issue by andreafalzetti** [PlatziDev/socket.io-react#1]
Hi, I'm trying to integrate this package into my React app.

I'm using webpack and I get this error when I build:

```
ERROR in my-app.0.0.1.js from UglifyJs
SyntaxError: Unexpected token: name (SocketProvider)
```

Do you have any idea of the reason?

Gracias

**Comment by sergiodxa** [comment]
can you share a basic code to replicate the error?

**Comment by shisaq** [comment]
I met the same problem, here is the error info:
```
ERROR in main.min.js from UglifyJs
SyntaxError: Unexpected token: name (SocketProvider) [../~/socket.io-react/build/SocketProvider.js:21,0]
```

And this is the relevant code from **line 21** of `SocketProvider.js`:

```javascript
let SocketProvider = function (_Component) {
  _inherits(SocketProvider, _Component);

  function SocketProvider() {
    _classCallCheck(this, SocketProvider);

    return _possibleConstructorReturn(this, Object.getPrototypeOf(SocketProvider).apply(this, arguments));
  }

  _createClass(SocketProvider, [{
    key: 'getChildContext',
    value: function getChildContext() {
      return {
        socket: this.props.socket
      };
    }
  }, {
    key: 'render',
    value: function render() {
      return this.props.children;
    }
  }]);

  return SocketProvider;
}(_react.Component);
```

I'm not quite sure about the reason, is it because of `let`?

**Comment by shisaq** [comment]
I've tried to change `let` into `var`, then the error disappeared.

It looks no problem on SocketProvider, but I'm just afraid if it would go down because of my replace...

Anyway, I think the error is still from keyword `let`, I wish it can be fixed. Thanks!

**Comment by sergiodxa** [comment]
I just published the [v1.1.2](https://github.com/PlatziDev/socket.io-react/releases/tag/v1.1.2) with the `let` replaced with a `var`, let me know if the error continue :)
